### ISSUE TITLE
Remove failOrWarn task or rename it somehow

### ISSUE THREAD

**Issue by vlsi** [autonomousapps/dependency-analysis-android-gradle-plugin#276]
**Is your feature request related to a problem? Please describe.**

The issue is that build failure should be understandable and actionable even for those who have no idea what Gradle is.

Build failure like `FAILURE: Build failed with an exception, Execution failed for task ':failOrWarn'.` would be puzzling and annoying, especially for those who are not very eager to learn Gradle, plugins, and so on.

**Describe the solution you'd like**

What if `failOrWarn` task was removed or replaced with a task with a human-understandable name?
Is there a reason `failOrWarn` is different from `aggregateAdvice`?

In other words, I launched `buildHealth`, and it passed, however, `failOrWarn` failed for an unknown reason. That was not very expected for me.

Note: re-execution of `failOrWarn` produces even more obscure failure as it prints no details to the console except the links to json files.

**Describe alternatives you've considered**

No idea :-/

**Additional context**

https://github.com/autonomousapps/dependency-analysis-android-gradle-plugin/issues/275

**Comment by autonomousapps** [comment]
Thank you for the request.

**Comment by autonomousapps** [comment]
I'm going to address this as part of https://github.com/autonomousapps/dependency-analysis-android-gradle-plugin/issues/275. As such, closing this as a duplicate.
